view/dispense.py

```diff
@@ -11,7 +11,7 @@
 VIEW_DISPENSE = Blueprint('dispense', __name__, url_prefix='/dispense')
 
 @VIEW_DISPENSE.route('/<pid>', methods=('POST',))
-def by_project_index(pid: str) -> str | ResponseBase:
+def by_project_index(pid: str) -> str | ResponseBase | Response:
```

Comment:
調整 L47 這裡就不用加 `Response`
```suggestion
def by_project_index(pid: str) -> str | ResponseBase:
```

Reply:
感謝建議～